machine_manager/README.rst

```diff
@@ -0,0 +1,27 @@
+Machine Manager
```

Comment:
Module name suggestion: equipment_manager

At least for OCA module

It could be use not only for manufacturing (there is no dependency on mrp):
- network material/device (I saw mac address field)
- manual tools
- machine
- vehicle
- etc.